src/libraries/System.Text.Json/gen/ContextGenerationSpec.cs

```diff
@@ -1,46 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using System.Collections.Generic;
-using System.Text.Json.Serialization;
-using System.Text.Json.Reflection;
-using System.Diagnostics;
-using Microsoft.CodeAnalysis;
```

Comment:
This file has been moved under the `Model` folder, but git seems to think it's been deleted.